core/src/main/java/org/bitcoinj/core/NetworkParameters.java

```diff
@@ -505,4 +500,23 @@ public int getMajorityWindow() {
         }
         return flags;
     }
+
+    public abstract int getProtocolVersionNum(final ProtocolVersion version);
+
+    public static enum ProtocolVersion {
+        MINIMUM(70000),
+        PONG(60001),
+        BLOOM_FILTER(70000),
+        CURRENT(70001);
+
+        private int bitcoinProtocol;
+
+        ProtocolVersion(final int setNum) {
```

Comment:
"setNum" feels like a weird name for a method parameter. Maybe "bitcoinProtocol"?